core/src/main/java/org/bitcoinj/governance/GovernanceVote.java

```diff
@@ -341,19 +341,19 @@ protected void bitcoinSerializeToStream(OutputStream stream) throws IOException
 
     public boolean isValid(boolean useVotingKey) {
         if (nTime > Utils.currentTimeSeconds() + (60 * 60)) {
-            log.info("gobject--CGovernanceVote::IsValid -- vote is too far ahead of current time - %s - nTime %lli - Max Time %lli\n", getHash().toString(), nTime, Utils.currentTimeSeconds() + (60 * 60));
+            log.info("gobject--CGovernanceVote::IsValid -- vote is too far ahead of current time - {} - nTime {} - Max Time {}", getHash().toString(), nTime, Utils.currentTimeSeconds() + (60 * 60));
```

Comment:
the string used in the log could be shorted, but the Governance classes are not used by Dash Wallet.